x-pack/plugin/security/qa/security-trial/src/javaRestTest/java/org/elasticsearch/xpack/security/apikey/ApiKeyRestIT.java

```diff
@@ -241,8 +287,16 @@ public void testGrantTargetCanUpdateApiKey() throws IOException {
         assertThat(apiKeyEncoded, not(emptyString()));
 
         doTestUpdateApiKey(apiKeyName, apiKeyId, apiKeyEncoded, null);
+        // bulk update also allowed
+        final var bulkUpdateApiKeyRequest = new Request("POST", "_security/api_key/_bulk_update");
+        final Map<String, Object> updateApiKeyRequestBody = Map.of("ids", List.of(apiKeyId));
+        bulkUpdateApiKeyRequest.setJsonEntity(
+            XContentTestUtils.convertToXContent(updateApiKeyRequestBody, XContentType.JSON).utf8ToString()
+        );
+        assertOK(performRequestUsingRandomAuthMethod(bulkUpdateApiKeyRequest));
```

Comment:
I went with more explicit calls here because the test suite seems to lean more towards explicit calls and less randomization. It's a little easier to see at a glance what the test actually does that way. But I'm also good with randomizing inside `doTestUpdateApiKey`, or, to keep it more explicit and a `doTestBulkUpdateApiKey` and randomize the call here instead:

`randomBoolean() ? doTestUpdateApiKey() : doTestBulkUpdateApiKey()`